enocean/README.md

```diff
@@ -132,29 +132,35 @@ EnOcean_Item:
                 enocean_rx_key: BI
     
     Brightness_Sensor:
-                name: brightness_sensor_east
-                remark: Eltako FAH60
-                type: num
-                enocean_rx_id: 01A51DE6
-                enocean_rx_eep: A5_06_01
-                enocean_rx_key: BRI
-                visu_acl: rw
-                sqlite: 'yes'
+        name: brightness_sensor_east
+        remark: Eltako FAH60
```

Comment:
AFAIK, you can use `comment` setting to add some comments to the YAML file, instead of introducing a new `remark` setting.

Reply:
I saw the remark in an example, but comment is also ok for me.
Changed all "remark" into "comment"

Reply:
Hi ohinckel,
were is the information that "comment" is the right attribute of the item?
If you look in the latest documentation there is described to use the attribute remark!